src/lib/stores/billing.ts

```diff
@@ -183,18 +183,23 @@ export function checkForTrialEnding(org: Organization) {
     }
 }
 
-export function checkForUsageLimit(org: Organization) {
+export async function checkForUsageLimit(org: Organization) {
     if (!org?.billingLimits) {
         readOnly.set(false);
         return;
     }
     const { bandwidth, documents, executions, storage, users } = org.billingLimits;
+    const members = await sdk.forConsole.teams.listMemberships(org.$id);
+    const plan = get(plansInfo).plans.find((plan) => plan.$id === org.billingPlan);
```

Comment:
Can we store plansInfo as a Map instead of an Array so that it's easier to lookup stuff? If so, we can definitely do this later rather than now.

Reply:
Makes sense as a future refactor 👍